monkey/infection_monkey/monkey.py

```diff
@@ -264,8 +260,9 @@ def _setup_agent_event_serializers(self) -> AgentEventSerializerRegistry:
         return agent_event_serializer_registry
 
     def _build_server_list(self, relay_port: int):
+        my_servers = [str(s) for s in self._opts.servers]
         relay_servers = [f"{ip}:{relay_port}" for ip in get_my_ip_addresses()]
```

Comment:
Building command line and relay server socket addresses should be separate. Maybe we'll need relay socket addresses for something else one day.

Reply:
I'm no fortune teller, but I think we're safe leaving this code as is. I don't anticipate the code to build these server lists to change, so it should be trivial to break that functionality out if and when we need it.